APE18.rst

```diff
@@ -0,0 +1,70 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+APE 10 previously determined supported CPython versions, and provided a list up to the 4.0 release.
+Since APE 10 was written the wider scientific Python community have written and largely adopted `NEP 29`_ as a policy for supported versions of both CPython and Numpy.
+This APE proposes that Astropy adopt this policy following other projects in the scientific ecosystem.
+
+
+Detailed description
+--------------------
+
+This APE fully adopts `NEP 29`_ for both CPython and Numpy versions for the core astropy package, and it supersedes APE 10 wherever there is a conflict between that APE and this APE or `NEP 29`_.
+
+This APE will not replicate the content of `NEP 29`_, and the full text of `NEP 29`_ is adopted via this APE.
+In addition to the content of `NEP 29`_, the APE also extends the numpy policy to all of the core package dependencies.
+This leads to the following support policy for dependencies of the core package.
+
+Astropy core will support:
+
+* All minor versions of Python released 42 months prior to a non-bugfix, and at minimum the two latest minor versions.
+* All minor versions of numpy released in the 24 months prior to a non-bugfix, and at minimum the last three minor versions.
+* Versions of other runtime dependencies released 24 months prior to a non-bugfix release.
+
+These versions of the dependencies are to be supported for bugfix lifetime of the given astropy version.
+
+This version support policy allows, at the discretion of the maintainers, to adopt the supported version of the next release in situations where it is judged to be valuable.
+For example, this could be done in the case of an important change in numpy or a change over date very close to the release.
+
+
+Implementation
+--------------
+
+Some initial work has been done in `#10900 <https://github.com/astropy/astropy/pull/10900>`__ on bumping the minimum Python version in the core astropy package to Python 3.7+.
+
```

Comment:
The CoCo discussed this APE just now and we agree it is good to go but wanted to make a small suggestion to mention the release calendar as follows:

```suggestion
Some initial work has been done in `#10900 <https://github.com/astropy/astropy/pull/10900>`__ on bumping the minimum Python version in the core astropy package to Python 3.7+.

The release calendar for the astropy core package will be updated to include the minimum Python and Numpy version for each future release.
```

We didn't want to include the link since it's just a wiki page which may move in future. Does this seem reasonable?